From 9a1fa7e13b5feefb76902287ae7ca2554d18cf15 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 18 Mar 2025 21:01:15 +0100
Subject: [PATCH 05/11] Revert "ci: Disable container building temporarily"

gitlab.freedesktop.org is back, so we can re-enable the
jobs that depend on it.

This reverts commit ad88a6e715bf751488d796d717aa83cafff01d65.

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/4343>
---
 .gitlab-ci.yml | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml
index 5e33496d0..4a0015a1a 100644
--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -326,14 +326,9 @@ check-merge-request:
   rules:
     - !reference [.only-merge-requests, rules]
 
-.dummy-container-build:
-  script:
-    - echo Dummy container build
-
 build-fedora-container@x86_64:
   extends:
-    #- .fdo.container-build@fedora@x86_64
-    - .dummy-container-build
+    - .fdo.container-build@fedora@x86_64
     - .mutter.fedora@x86_64
     - .mutter.skip-git-clone
   stage: prepare
@@ -342,8 +337,7 @@ build-fedora-container@x86_64:
 
 build-fedora-container@aarch64:
   extends:
-    #- .fdo.container-build@fedora@aarch64
-    - .dummy-container-build
+    - .fdo.container-build@fedora@aarch64
     - .mutter.fedora@aarch64
     - .mutter.skip-git-clone
   stage: prepare
-- 
2.47.1

